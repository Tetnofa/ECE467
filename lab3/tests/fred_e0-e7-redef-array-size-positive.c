int a1[4];
int a1[-3];

int a2[-3];
int a2[1];
