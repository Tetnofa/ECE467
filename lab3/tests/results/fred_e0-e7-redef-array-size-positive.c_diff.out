0a1,9
> Program [useIO=0](1,0)
> 	Array Declaration	Int[4](1,0)
> 		Identifier[name:a1](1,4)
> 	Array Declaration	Int[-3](2,0)
> 		Identifier[name:a1](2,4)
> 	Array Declaration	Int[-3](4,0)
> 		Identifier[name:a2](4,4)
> 	Array Declaration	Int[1](5,0)
> 		Identifier[name:a2](5,4)
